esignet-ida115-integration-impl/src/main/java/io/mosip/ida115/esignet/integration/service/Ida115AuthenticatorImpl.java

```diff
@@ -591,9 +591,9 @@ private void addFormattedSubAttributes(List<String> idSchemaAttributes, Map<Stri
 		//String trimmedValue = identityInfoValueStr.substring(0, identityInfoValueStr.lastIndexOf(addressValueSeparator));
 		attributesMap.put(FORMATTED + localeAppendValue, identityInfoValue.toString());
 		if (langCodeFound && addLocale) {
-			respMap.put(consentedAttributeName + localeAppendValue, (consentedAttributeName.equals(consentedAddressAttributeName)) ? attributesMap : identityInfoValue.toString());
+			respMap.put(consentedAttributeName + localeAppendValue, (consentedAttributeName.equals(consentedAddressAttributeName) || consentedAttributeName.equals(consentedNameAttributeName)) ? attributesMap : identityInfoValue.toString());
 		} else {
-			respMap.put(consentedAttributeName, (consentedAttributeName.equals(consentedAddressAttributeName)) ? attributesMap : identityInfoValue.toString());
+			respMap.put(consentedAttributeName, (consentedAttributeName.equals(consentedAddressAttributeName) || consentedAttributeName.equals(consentedNameAttributeName)) ? attributesMap : identityInfoValue.toString());
```

Comment:
attributesMap should only be set to the "address" claim, "name" claim must always be a string as per SPEC. Please remove this new OR condition